department-of-veterans-affairs/va.gov-cms
# VACMS 9579 QA Reusability on R&S Pages, with keyword filter and description field added to Q&A Groups

## Description

Relates to #9737, #8630, #9579 

## Testing done
By hand: added and edited content, reviewed other content. CMS/backend side only.

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As any user with editor privileges, add new or edit existing Resources and Support content > Add main content > Q&A Group, you should be able to:
- filter by section and/or title/keyword
- select one or more Questions
- save 

Ensure that editors still have correct privileges for this and that others do not

### Definition of Done

- [ ] Frontend needs?
- [ ] Design approval

### Select Team for PR review



### Is this PR blocked by another PR?

- [ ] Find out if FE work needs to go first

### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
